nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/bulletins/ui/bulletin-board/bulletin-board-list/bulletin-board-list.component.ts

```diff
@@ -143,12 +143,12 @@ export class BulletinBoardList implements AfterViewInit {
     getSeverity(severity: string) {
         switch (severity.toLowerCase()) {
             case 'error':
-                return 'bulletin-error';
+                return 'bulletin-error material-warn-darker';
```

Comment:
I can't seem to find where "material-warn-darker" is defined.

Reply:
good catch. it should probably also be `mat-warn-darker` anyhow.

Reply:
Good catch guys!